### PR TITLE
Add 2021-09-07 geo issues

### PR DESCRIPTION
**Pull request by jklukas**
See https://bugzilla.mozilla.org/show_bug.cgi?id=1729069

### CODE REVIEW

File: backfill/2021-09-07-error-geo/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+Permissions issues when trying to set up tables in the backfill project:
+
+```
++ bq query --nouse_legacy_sql 'CREATE OR REPLACE TABLE `moz-fx-data-backfill-6.contextual_services_live.quicksuggest_click_v1` LIKE `moz-fx-data-shared-prod.contextual_services_live.quicksuggest_click_v1`'
+BigQuery error in query operation: Error processing job 'moz-fx-data-shared-prod:bqjob_r6228d0d1b75d8b59_0000017bc16718cb_1': Access
+Denied: Table moz-fx-data-shared-prod:contextual_services_live.quicksuggest_click_v1: User does not have permission to query table
+moz-fx-data-shared-prod:contextual_services_live.quicksuggest_click_v1.
+```
+
+My own user can't access the contextual_services tables, or the regrets_reporter table in telemetry.
```
Comment (jklukas) [inline backfill/2021-09-07-error-geo/README.md:237]: cc @whd

Comment (whd) [reply backfill/2021-09-07-error-geo/README.md:237]: I've run the script filtering to `contextual_services_live`, `regrets_reporter_live` and `telemetry_live` (filtered to xfocsp/regrets tables).

File: backfill/2021-09-07-error-geo/README.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    --bqReadMethod=storageapi \
+    --outputType=bigquery \
+    --bqRowRestriction=\"CAST(submission_timestamp AS DATE) BETWEEN '2021-08-20' AND '2021-09-03'\"
+    --bqWriteMethod=file_loads \
+    --bqClusteringFields=submission_timestamp \
+    --output=${PROJECT}:\${document_namespace}_live.\${document_type}_v\${document_version} \
+    --errorOutputType=bigquery \
+    --errorOutput=${PROJECT}:payload_bytes_error.${PIPELINE_FAMILY} \
+    --experiments=shuffle_mode=service \
+    --region=us-central1 \
```
Comment (whd) [inline backfill/2021-09-07-error-geo/README.md:50]: I don't think this is what caused the quota issue but we generally use us-west1 for workloads. If we had quotas bumped for these projects from e.g. the initial AWS import they may have only applied to us-west1.

File: backfill/2021-09-07-error-geo/README.md [hunk 3]
```diff
@@ -0,0 +1,325 @@
...
+bq ls --project_id=moz-fx-data-backfill-7 | tail -n+3 | awk '{print $1}' | xargs -I{} -n1 echo bq rm -r -f "moz-fx-data-backfill-7:{}"
+gsutil ls -p moz-fx-data-backfill-7 | xargs echo gsutil -m rm -r
+bq ls --project_id=moz-fx-data-backfill-8 | tail -n+3 | awk '{print $1}' | xargs -I{} -n1 echo bq rm -r -f "moz-fx-data-backfill-8:{}"
+gsutil ls -p moz-fx-data-backfill-8 | xargs echo gsutil -m rm -r
+bq ls --project_id=moz-fx-data-backfill-17 | tail -n+3 | awk '{print $1}' | xargs -I{} -n1 echo bq rm -r -f "moz-fx-data-backfill-17:{}"
+```
+
+An admin needs to run:
+```
+bq rm moz-fx-data-shared-prod:payload_bytes_raw.stub_installer_snapshot_bug1729069
```
Comment (jklukas) [inline backfill/2021-09-07-error-geo/README.md:219]: I'm ready for @jasonthomas to run this command when convenient. I've cleaned up all the tables in `moz-fx-data-backfill-6` that I have permissions to delete.

Comment (jasonthomas) [reply backfill/2021-09-07-error-geo/README.md:219]: Done.

**Pull request merged**